spec/models/async_transaction/vet360/address_transaction_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
Perhaps delete this file, since we don't have any specs for it yet?  And then we can recreate when needed?